dist/dorsal.js

```diff
@@ -207,7 +207,15 @@ DorsalCore.prototype._runPlugin = function(el, pluginName) {
  * @return {Array} registered plugin names
  */
 DorsalCore.prototype.registeredPlugins = function() {
-    return Object.keys(this.plugins);
+    var pluginKeys = Object.keys(this.plugins);
+
+    if (!pluginKeys.length) {
+        if (console && console.warn) {
```

Comment:
we already use console.log in a controlled way https://github.com/eventbrite/dorsal/blob/master/src/log.js#L48-L59